### ISSUE TITLE
Required param ID is redundant??

### ISSUE THREAD

**Issue by rjnl** [mackensen/moodle-report_rubric_list#7]
Report shows all rubrics a user has created in all courses. 
So it does not matter from which course a user is accessing this  report. This makes the course id required parameter somewhat redundant.

**Comment by mackensen** [comment]
Yes and no. It's a little awkward having a global report living in the course context, but that's simplest for navigation. I think I need the course id in order to maintain that navigation context.
